classes/class-woothemes-sensei-certificates.php

```diff
@@ -1480,7 +1480,7 @@ public function email_certificate_link( $template ) {
 	public function enqueue_block_editor_assets() {
 		$screen = get_current_screen();
 
-		if ( $screen && 'page' === $screen->post_type ) {
+		if ( $screen && 'page' === $screen->post_type || 'course' === $screen->post_type ) {
```

Comment:
You can ignore this file, since I just run the formatter for JS and it was updated.

Reply:
I think this comment was supposed to be in the other file, right?

Reply:
You suppose right haha